<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Post Venta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PostVenta.qrc">
    <normaloff>:/images/Toolbox.png</normaloff>:/images/Toolbox.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCommandLinkButton" name="PantallaIngreso">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>90</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="PostVenta.qrc">
      <normaloff>:/images/Box-icon.png</normaloff>:/images/Box-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="PantallaTrabajos">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>90</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="PostVenta.qrc">
      <normaloff>:/images/delivery-box-icon.png</normaloff>:/images/delivery-box-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="PantallaReparaciones">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>90</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="PostVenta.qrc">
      <normaloff>:/images/MartilloyDestornillador.png</normaloff>:/images/MartilloyDestornillador.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingreso</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modulos de trabajo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trabajos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reparaciones</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="PantallaBusqueda">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="PostVenta.qrc">
      <normaloff>:/images/Analisis.png</normaloff>:/images/Analisis.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Busqueda</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Puerto</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionBorrar"/>
    <addaction name="actionQuit"/>
    <addaction name="actionArchivos"/>
   </widget>
   <widget class="QMenu" name="menuDataBases">
    <property name="title">
     <string>DataBases</string>
    </property>
    <addaction name="actionEditar"/>
    <addaction name="actionActualizar"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuDataBases"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="actiondbEdit"/>
   <addaction name="actionActualizar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;Acerca</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onectar</string>
   </property>
   <property name="toolTip">
    <string>Conectar puerto serie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Desconectar</string>
   </property>
   <property name="toolTip">
    <string>Desconectar puerto serie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configurar</string>
   </property>
   <property name="toolTip">
    <string>Configurar puerto seire</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Borrar</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/Save-icon.png</normaloff>:/images/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actiondbEdit">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/database-edit-icon.png</normaloff>:/images/database-edit-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;dbEdit</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionBorrar">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </action>
  <action name="actionActualizar">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/database-refresh-icon.png</normaloff>:/images/database-refresh-icon.png</iconset>
   </property>
   <property name="text">
    <string>Abrir db</string>
   </property>
   <property name="toolTip">
    <string>Abrir db</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset resource="PostVenta.qrc">
     <normaloff>:/images/database-edit-icon.png</normaloff>:/images/database-edit-icon.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="actionArchivos">
   <property name="text">
    <string>Archivos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PostVenta.qrc"/>
 </resources>
 <connections/>
</ui>
